src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
```

Comment:
Body element is actually available on the document as property, no need for using the querySelector:
```js
const body = document.body;
```

Reply:
All right! Thank you for comments :)